table/block_based/cachable_entry.h

```diff
@@ -119,6 +119,25 @@ class CachableEntry {
     ResetFields();
   }
 
+  void TransferTo(PinnableSlice* slice) {
+    if constexpr (std::is_same_v<std::string, T> ||
+                  std::is_same_v<std::string_view, T> ||
+                  std::is_same_v<PinnableSlice, T> ||
+                  std::is_same_v<Slice, T> || std::is_same_v<char, T>) {
```

Comment:
Maybe remove these for now and keep only the case for `std::string` because that's what we need. Or do you see the other cases will be needed in the future?

Reply:
Sure. We probably can keep `char*` if we plan to replace `std::string` with it in the future.

Reply:
I got rid of them to make it cleaner. Thanks.